SPARCore/GADArticle.m

```diff
@@ -1,7 +1,107 @@
 #import "GADArticle.h"
+#import "GADRemoteModel.h"
 
 @implementation GADArticle
 
+static NSString *const API_ARTICLE_ID = @"id";
+static NSString *const API_AUTHORS = @"authors";
+static NSString *const API_BRIEF = @"brief";
+static NSString *const API_CONTENT = @"content";
+static NSString *const API_DATE_PUBLISHED = @"datePublished";
+static NSString *const API_HEADER_IMAGE = @"headerImage";
+static NSString *const API_PUBLICATION_ID = @"publication";
+static NSString *const API_TITLE = @"title";
+
+static NSString *const API_HOSTNAME = @"https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/";
+static NSString *const API_PREFIX = @"devstable";
+static NSString *const API_PUBLICATION_PATH = @"publications";
+static NSString *const API_ARTICLE_PATH = @"articles";
+
++ (void) articlesFromPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADArticle *>
+                                                                           *_Nullable articles,
+                                                            NSError *_Nullable error))completion {
+    NSURL *queryURL = [GADArticle createURLWithPublication:publicationId];
+    
+    [GADRemoteModel fetchModelsWithParams:queryURL
+                          queryParameters:@{}
+                         modelTransformer:^(NSData *jsonData) {
+                            return [GADArticle articlesFromJSON:jsonData];
+                         }
+                        completionHandler:completion];
+
+}
+
+- (void) populateArticleWithId: (void(^_Nonnull)(GADRemoteModel * *_Nullable model,
```

Comment:
Taken care of. We didn't find this method in the header file and didn't find it called anywhere. Do we need to change anything else for this?